src/NECompletion/ReverseAlphabeticSorter.class.st

```diff
@@ -2,19 +2,19 @@
 Reverse alphabetic sorting
 "
 Class {
-	#name : #TestReverseAlphabeticSorter,
-	#superclass : #TestSorter,
+	#name : #ReverseAlphabeticSorter,
+	#superclass : #CompletionSorter,
 	#category : #'NECompletion-New'
 }
```

Comment:
Not related to this particular changes but I am curious did you see SortFunction's in Pharo?
It looks like it can be used here instead of new sorters hierarchy. Check class comment and subclasses.

Reply:
Ah, interesting... I put it on the TODO to check that